modules/ding_ekurser/ding_ekurser.install

```diff
@@ -43,3 +43,11 @@ function ding_ekurser_uninstall() {
     variable_set('search_active_modules', $search_modules);
   }
 }
+
+/**
+ * Implement hook_update_N.
```

Comment:
The "implement" statement should not be part of the comment.

Update hooks should only have a single line as comment with a short description.